kafka-connect/src/main/scala/net/manub/embeddedkafka/connect/EmbeddedConnectConfig.scala

```diff
@@ -0,0 +1,46 @@
+package net.manub.embeddedkafka.connect
+
+import java.nio.file.Path
+
+import net.manub.embeddedkafka.EmbeddedKafkaConfig
+
+import org.apache.kafka.connect.runtime.standalone.StandaloneConfig
+import org.apache.kafka.connect.runtime.WorkerConfig
+
+/**
+  * Kafka Connect config generator.
+  *
+  * @tparam C an [[EmbeddedKafkaConfig]]
+  */
+private[embeddedkafka] trait EmbeddedConnectConfig[C <: EmbeddedKafkaConfig] {
+  protected[embeddedkafka] def baseConnectConfig(
+      connectPort: Int,
+      offsets: Path
+  )(
+      implicit kafkaConfig: C
+  ): Map[String, String] =
+    Map(
+      WorkerConfig.LISTENERS_CONFIG                        -> s"http://localhost:$connectPort",
+      WorkerConfig.BOOTSTRAP_SERVERS_CONFIG                -> s"localhost:${kafkaConfig.kafkaPort}",
+      WorkerConfig.KEY_CONVERTER_CLASS_CONFIG              -> "org.apache.kafka.connect.json.JsonConverter",
+      WorkerConfig.VALUE_CONVERTER_CLASS_CONFIG            -> "org.apache.kafka.connect.json.JsonConverter",
+      WorkerConfig.OFFSET_COMMIT_INTERVAL_MS_CONFIG        -> "10000",
+      StandaloneConfig.OFFSET_STORAGE_FILE_FILENAME_CONFIG -> offsets.toFile.getAbsolutePath
```

Comment:
`.toFile` conversion could be skipped by calling `.toAbsolutePath.toString`

Reply:
Done!